### PR TITLE
Web server logs

### PR DESCRIPTION
**Pull request by NoaAzoulay**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
added webserver_log check

### CODE REVIEW

**Review by arielkru — APPROVED** [review]
Looks great :)

File: tests/terraform/checks/resource/aws/test_MWAAWebserverLogsEnabled.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import unittest
+from pathlib import Path
+
+from checkov.terraform.runner import Runner
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.checks.resource.aws.MWAAWebserverLogsEnabled import check
+
+
+class MWAAWebserverLogsEnabled(unittest.TestCase):
```
Comment (arielkru) [inline tests/terraform/checks/resource/aws/test_MWAAWebserverLogsEnabled.py:9]: The convention is to add the prefix `Test` to the test classes so this should be `TestMWAAWebserverLogsEnabled`

Comment (NoaAzoulay) [reply tests/terraform/checks/resource/aws/test_MWAAWebserverLogsEnabled.py]: thanks :) i changed it

**Review by itai1357 — COMMENTED** [review]
Great!!

**Pull request merged**